addons/l10n_fr_account/i18n/fr.po

```diff
@@ -967,29 +967,31 @@ msgstr "Opérations imposables (H.T.)"
 #. module: l10n_fr_account
 #: model_terms:ir.ui.view,arch_db:l10n_fr_account.fec_export_wizard_view
 msgid "Technical Info"
-msgstr ""
+msgstr "Informations techniques"
 
 #. module: l10n_fr_account
 #: model_terms:ir.ui.view,arch_db:l10n_fr_account.fec_export_wizard_view
 msgid "Technical Name"
-msgstr ""
+msgstr "Nom technique"
 
 #. module: l10n_fr_account
 #: model:ir.model.fields,field_description:l10n_fr_account.field_l10n_fr_fec_export_wizard__test_file
 msgid "Test File"
-msgstr ""
+msgstr "Fichier de test"
 
 #. module: l10n_fr_account
 #: model_terms:ir.ui.view,arch_db:l10n_fr_account.fec_export_wizard_view
 msgid ""
 "The encoding of this text file is UTF-8. The structure of file is CSV "
 "separated by pipe '|'."
 msgstr ""
+"L'encodage de ce fichier texte est UTF-8. La structure du fichier est CSV, "
+"séparée par un caractère pipe « | »."
```

Comment:
The whole sentence sounds a bit odd to me, but I have a feeling that the source terms weren't really good to begin with. "Séparée" refers to "la structure". Being separated by pipes *is* the structure itself, so saying the structure is separated by pipes sounds wrong to me. Also, I find it amusing that a format called "CSV" (**Comma**-Separated Values) could use anything other than a comma as a separator 😄